autoscale-scaler-marathon/src/main/java/com/hpe/caf/autoscale/scaler/marathon/AppInstancePatcher.java

```diff
@@ -29,11 +29,15 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.Arrays;
+import java.util.logging.Level;
 import org.apache.http.client.utils.URIBuilder;
 import org.apache.http.message.BasicNameValuePair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;

```

Comment:
We are adding both jul and slf4j logging imports?  Are we using two logging frameworks?